modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_forms.scss

```diff
@@ -90,6 +90,10 @@
 	}
 }
 
+.input-group .form-validator-stack {
+	order: 1;
```

Comment:
I think we can increase the order value to be higher maybe 10? This makes `help-block` less prone to breaking if someone is customizing the order values of `input-group-addon` or `input`.

Also we can add `width: 100%;`, long text with no whitespace doesn't break to new line.